ext/AvizExt/viz/taxa_dynamics.jl

```diff
@@ -187,12 +200,16 @@ function taxonomy_by_intervention!(
 
     # Plot and calculate confidence intervals
     confints = zeros(n_timesteps, n_functional_groups, 3)
-    for (idx, taxa) in enumerate(relative_taxa_cover.taxa)
+    for (idx, taxa) ∈ enumerate(relative_taxa_cover.taxa)
         confints[:, idx, :] = series_confint(relative_taxa_cover[taxa=At(taxa)])
-        show_confints ? band!(
-            ax, 1:n_timesteps, confints[:, idx, 1], confints[:, idx, 3];
-            color=(colors[idx], 0.4)
-        ) : nothing
+        if show_confints
```

Comment:
not sure I like it changing if/else to elvis operator. what are your thoughts?

Reply:
I think It should leave these alone. If I use the ternary operator, it's for a reason, and vice versa.